client/app/cdn/dedicated/manage/logs/cdn-dedicated-manage-logs.html

```diff
@@ -0,0 +1,32 @@
+<div data-ng-controller="CdnLogsCtrl as listCtrl">
+
+    <ovh-tail-logs>
+        <div class="row ml-0"
+             data-ng-repeat="log in listCtrl.logger.logs track by $index"
+             data-ng-class="{
+                'text-primary' : log.message.status >= 200 && log.message.status < 300,
+                'text-warning' : log.message.status >= 300 && log.message.status < 499,
+                'text-danger'  : log.message.status > 499
+            }">
+
+            <div class="col-md-12">
+                <span data-ng-bind="log.message.source" title="ip_source" />
```

Comment:
Avoid using `title`, it can't be seen by screen-readers and mobile devices

Reply:
`span` require an ending tag `<span></span>`, even if it's rempty

Reply:
I understand what you are trying to do, you'd better use https://developer.mozilla.org/en-US/docs/Web/HTML/Element/dl instead :)